shell/indicators/StorageIndicator.qml

```diff
@@ -32,68 +32,35 @@ import Hawaii.Hardware 1.0 as Hardware
 import ".."
 
 Indicator {
-    name: "storage"
+    name: qsTr("Storage")
```

Comment:
name is not translatable like text, it's just the name to reference the indicator